conanfile.py

```diff
@@ -99,9 +99,7 @@ def requirements(self):
 
         if self.options.with_gui:
             self.requires("freetype/2.10.0@bincrafters/stable#0")
-            self.requires(
-                "freetype-gl/8d9a97a@{}#2836d28f3d91c308ec9652c2054015db".format(self._orbit_channel))
-            self.requires("glew/2.1.0@{}#0".format(self._orbit_channel))
+            self.requires("freetype-gl/79b03d9@{}".format(self._orbit_channel))
```

Comment:
I'd have expected glad to be added here.

Reply:
Hmm, it is added in the freetyp conanfile.py below. I guess this makes glad available in OrbitGL as well, right?

Reply:
Yes, you're right. it works by accident because freetype-gl requires GLAD. But we really should add it here in the include-what-you-use manner. Will fix...

Reply:
Done.